Database/creazione_database.sql

```diff
@@ -1,83 +1,281 @@
+)
+
+-- Crea la tabella Destinazione
+
 DROP TABLE IF EXISTS Destinazione;
 
 CREATE TABLE Destinazione(
-	id_destinazione		INT PRIMARY KEY,
-	nome				VARCHAR(20) NOT NULL,
-	cognome				VARCHAR(20) NOT NULL,
-	numero_telefonico	VARCHAR(15),
-	CAP					VARCHAR(5) NOT NULL,
-	via 				VARCHAR(15) NOT NULL,
-	numero_civico		SMALLINT NOT NULL,
-	utente 				VARCHAR(20) NOT NULL,	
-	FOREIGN KEY(utente) REFERENCES Utente(username) ON DELETE CASCADE
+	id_destinazione			INT PRIMARY KEY,
+	nome_cognome				VARCHAR(40) NOT NULL,
+	numero_telefonico		VARCHAR(15),
+	CAP									VARCHAR(5) NOT NULL,
+	via 								VARCHAR(15) NOT NULL,
+	numero_civico				SMALLINT NOT NULL,
+	utente 							VARCHAR(20) NOT NULL,
+	FOREIGN KEY (utente) REFERENCES Utente(username) ON DELETE CASCADE
 );
 
+-- Inserimento dati nella tabella Destinazione
+
+INSERT INTO Destinazione (id_destinazione, nome_cognome, numero_telefonico, CAP, via, numero_civico, utente) VALUES (
+(201, 'Utente Generico', '049XXXXXXX', '35100', 'Aldo Moro', 21, 'utente'),
+(202, 'Utente Generico', '049XXXXXXX', '35100', 'Ugo Bassi', 17, 'utente'),
+(203, 'User Uno', '346XXXXXXX', '35133', 'Don Stefani', 10, 'user1'),
+(204, 'User Due', '339XXXXXXX', '35142', 'Monte Bianco', 15, 'user2'),
+(205, 'User Tre', '333XXXXXXX', '35129', 'Andrea Palladio', 17, 'user3'),
+(206, 'User Quattro', '340XXXXXXX', '35122', 'Antonio Canova', 11, 'user4'),
+(207, 'User Cinque', '348XXXXXXX', '35100', 'Aldo Moro', 22, 'user5')
+)
+
+-- Crea la tabella Tabella Ordine
+
 DROP TABLE IF EXISTS Ordine;
 
 CREATE TABLE Ordine(
-	id_ordine		INT PRIMARY KEY,
+	id_ordine			INT PRIMARY KEY,
 	data_ordine		DATE NOT NULL,
-	data_consegna	DATE NOT NULL,
-	totale			FLOAT NOT NULL,
+	totale				FLOAT NOT NULL,
 	destinazione	INT NOT NULL,
-	FOREIGN KEY(destinazione) REFERENCES Destinazione(id_destinazione) ON DELETE NO ACTION
+	FOREIGN KEY (destinazione) REFERENCES Destinazione(id_destinazione) ON DELETE NO ACTION
 );
 
+-- Inserimento dati nella tabella Ordine
+
+INSERT INTO Ordine (id_ordine, data_ordine, totale, destinazione) VALUES (
+(301, '2019-02-19', 27.00, 201),
+(302, '2019-03-22', 28.00, 202),
+(303, '2019-04-11', 33.00, 203),
+(304, '2019-05-09', 31.50, 204),
+(305, '2019-05-07', 46.50, 205),
+(306, '2019-06-12', 40.00, 206),
+(307, '2019-07-22', 36.50, 207),
+(308, '2019-08-29', 66.00, 201),
+(309, '2019-10-16', 15.50, 202),
+(310, '2020-01-10', 42.50, 203),
+(311, '2020-01-13', 57.50, 204)
+)
+
+-- Crea la tabella Tabella Prodotto
+
 DROP TABLE IF EXISTS Prodotto;
 
 CREATE TABLE Prodotto(
-	nome 			VARCHAR(30) PRIMARY KEY,
-	categoria		ENUM('Antipasti','Primi Piatti','Teppanyako e tempure','Uramaki','Nigiri e Onigiri','Gunkan','Temaki','Hosomaki','Sashimi','Dessert') NOT NULL,
-	pezzi			TINYINT NOT NULL,
+	nome 				VARCHAR(30) PRIMARY KEY,
+	categoria		ENUM('Antipasti','Primi Piatti','Teppanyako e tempure','Uramaki','Nigiri','Gunkan','Temaki','Hosomaki','Sashimi','Dessert') NOT NULL,
```

Comment:
Come mai hai cancellato "e Onigiri"?

Reply:
> Chiarire cosa del nome e cognome separati

Nel senso prima era 'nome', 'cognome', ed ora ho messo 'nome cognome' in un attributo visto che nei form mi pare che non li chiediamo divisi.